### DISCUSSION TITLE
OceanGliders GitHub training

### DISCUSSION THREAD

**Discussion by soerenthomsen** [OceanGlidersCommunity/Oxygen_SOP#133] [Announcements]
**Next training**
Wednesday Nov 3, 9:00 - 9:45 CET

Interested: 
Estel Font (@EstelFont), Juliane Wihsgott (@juliane-w), Ben Blackledge (@BWBlack), Jasmin McInerney (@JasminMcIn), Chiara Monforte (@MOchiara), Johan Medholm (@jmedholm), Adi, have indicated interest so far.

Please post your GitHub handle if not mentioned above. So I can keep track. Thx

Here the meeting link: 
https://global.gotomeeting.com/join/235879493 

Here is our [sandbox playbox](https://github.com/OceanGlidersCommunity/LearningGitHub/blob/main/training_session_02.md)

**Comment by MOchiara** [comment]
Hi! I'm Chiara Monforte and I'd like to join the training

**Reply by soerenthomsen** [reply]
You find the meeting links above!

**Comment by jmedholm** [comment]
Hello! I'm Johan, and I too would like to join. [morganedholm](https://twitter.com/morganedholm)
